src/pe_structures.h

```diff
@@ -0,0 +1,517 @@
+/* Copyright 2017 - 2021 R. Thomas
+ * Copyright 2017 - 2021 Quarkslab
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// This is https://github.com/lief-project/LIEF/blob/master/include/LIEF/PE/structures.inc
```

Comment:
This file is taken almost as-is from the project indicated.
I chose the LIEF headers since they also use the apache license, and I figured that would be compatible enough.